project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,225 @@
+        env = cls.env
+
+        # Built-in project groups
+        group_project_user = env.ref("project.group_project_user")
+        group_project_manager = env.ref("project.group_project_manager")
+
+        # Sales group (fallback in case sale module is not installed)
+        try:
+            cls.group_sales_admin = env.ref("sale.group_sale_manager")
+        except ValueError:
+            cls.group_sales_admin = env["res.groups"].create(
+                {"name": "Sales / Manager (Test)"}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)